docs/pages/versions/unversioned/sdk/proximity.md

```diff
@@ -0,0 +1,168 @@
+      _unsubscribe();
+    } else {
+      _subscribe();
+    }
+  };
+
+  const _subscribe = () => {
+    this._subscription = Proximity.addListener(proximityData => {
+      setData(proximityData);
+    });
+  };
+
+  const _unsubscribe = () => {
+    this._subscription && this._subscription.remove();
+    this._subscription = null;
+  };
+
+  const { proximityState = false } = data;
+
+  return (
+    <View style={styles.sensor}>
+      <Text>Proximity:</Text>
+      <Text>Is close?: {proximityState ? "yes" : "no"}</Text>
+      <View style={styles.buttonContainer}>
+        <TouchableOpacity onPress={_toggle} style={styles.button}>
+          <Text>Toggle</Text>
+        </TouchableOpacity>
+      </View>
+    </View>
+  );
+}
+
+/* @hide const styles = StyleSheet.create({ ... }); */
+const styles = StyleSheet.create({
+  buttonContainer: {
+    flexDirection: 'row',
+    alignItems: 'stretch',
+    marginTop: 15,
+  },
+  button: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#eee',
+    padding: 10,
+  },
+  sensor: {
+    marginTop: 45,
+    paddingHorizontal: 10,
+  },
+});
+/* @end */
+```
+
+</SnackInline>
+
+## API
+
+```js
+import { Proximity } from 'expo-sensors';
+```
+
+## Methods
+
+### `Proximity.isAvailableAsync()`
+
+> You should always check the sensor availability before attempting to use it.
+
+Returns a promise which resolves into a boolean denoting the availability of the device proximity sensor.
+
+| OS      | Availability                |
+| ------- | --------------------------- |
+| iOS     | iOS 3+                      |
+| Android | Android ?.?+ (API Level ?+) |
+| Web     | `N/A`                       |
+
+#### Returns
+
+- A promise that resolves to a `boolean` denoting the availability of the sensor.
+
+### `Proximity.addListener((data: ProximityMeasurement) => void)`
+
+Subscribe for updates to the proximity sensor.
+
+```js
+const subscription = Proximity.addListener(({ proximityState }) => {
+  console.log({ proximityState });
+});
+```
+
+#### Arguments
+
+- **listener (_function_)** -- A callback that is invoked when a proximity sensor update is available. When invoked, the listener is provided a single argument that is an object containing: `proximityState: boolean` (_`isClose?`_).
+
+#### Returns
+
+- A subscription that you can call `remove()` on when you would like to unsubscribe the listener.
+
+### `Proximity.removeAllListeners()`
+
+Removes all listeners.
+
+## Types
+
+### `ProximityMeasurement`
+
+The proximity state returned from the native sensors.
+
+```typescript
+type ProximityMeasurement = {
+  proximityState: boolean;
+};
+```
+
+| Name             | Type                                         | Format     | iOS | Android | Web |
+| ---------------- | -------------------------------------------- | ---------- | --- | ------- | --- |
+| proximityState   | `boolean`                                    | `isClose?` | ✅  | ✅      | ❌  |
+
```

Comment:
I think "near" might be harder to misinterpret than "close".

Reply:
indeed. I can change it!

Reply:
Near would also be a bit closer to the W3 proposed spec https://w3c.github.io/proximity/#proximity-sensor-near